airbyte-api-server/src/main/openapi/api.yaml

```diff
@@ -0,0 +1,723 @@
+openapi: 3.0.2
+x-stoplight:
+  id: lo4tdzk9jxjfa
+info:
+  title: Airbyte API
+  version: 1.0.0
+paths:
+  "/connections/{connectionId}/sync":
+    post:
+      tags:
+        - connection
+        - RPC
+      responses:
+        "200":
+          description: Successful operation
+        "403":
+          $ref: "#/components/responses/NotAuthorizedResponse"
+        "404":
+          $ref: "#/components/responses/NotFoundResponse"
+      operationId: syncConnection
+      summary: Trigger a manual sync of the connection
+    #      security:
+    #        - bearerAuth: [ ]
+    parameters:
+      - name: connectionId
+        schema:
+          format: uuid
+          type: string
+        in: path
+        required: true
+      - name: X-Endpoint-API-UserInfo
+        in: header
+        schema:
+          type: string
+        description: an authorization header
+        required: false
+  #      - name: Authorization
+  #        in: header
+  #        schema:
+  #          type: string
+  ##          format: jwt
+  #        description: an authorization header
+  #        required: true
+  ##        type: string
+  "/connections/{connectionId}/reset":
+    post:
+      tags:
+        - connection
+        - RPC
+      responses:
+        "200":
+          description: Successful operation
+        "403":
+          $ref: "#/components/responses/NotAuthorizedResponse"
+        "404":
+          $ref: "#/components/responses/NotFoundResponse"
+      operationId: resetConnection
+      summary: Reset the data for the connection.
+      description: Reset the data for the connection. Deletes data generated by the connection in the destination. Resets any cursors back to initial state.
+    parameters:
+      - name: connectionId
+        schema:
+          format: uuid
+          type: string
+        in: path
+        required: true
+  /connections:
+    get:
+      tags:
+        - connection
+      parameters:
+        - name: identifiers
+          description: ""
+          schema:
+            $ref: "#/components/schemas/Identifier"
+          in: query
+        - name: workspaceId
```

Comment:
is there a reason this isn't required?

Reply:
is the idea that you can query by slug OR workspaceId OR identifiers? if so then should the docs say that explicitly? and then we throw the appropriate HTTP error code if that input is not provided.

Reply:
i think workspaceId should be required.